packages/cli-lib/github.js

```diff
@@ -124,8 +133,17 @@ async function cloneGitHubRepo(dest, type, repoName, sourceDir, options = {}) {
   return success;
 }
 
-async function getGitHubRepoContentsAtPath(repoName, path) {
-  const contentsRequestUrl = `https://api.github.com/repos/HubSpot/${repoName}/contents/${path}`;
+async function getGitHubRepoContentsAtPath(
+  repoName,
+  path,
+  customRepoPath = ''
+) {
+  let contentsRequestUrl;
+  if (customRepoPath) {
+    contentsRequestUrl = `https://api.github.com/repos/${customRepoPath}/contents/${path}`;
+  } else {
+    contentsRequestUrl = `https://api.github.com/repos/HubSpot/${repoName}/contents/${path}`;
+  }
```

Comment:
nit: Would we lose anything if we killed `isCustomRepoPath` entirely and used `HubSpot/hubspot-project-components` as the default repo value? That would let you work with the repo name/path as a single value and avoid adding conditional logic when you access the value, since both will follow the same structure of `org/repo`.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/837/commits/df3bddabd1b75142b473f1a965ca68a8dbebae97 and https://github.com/HubSpot/hubspot-cli/pull/837/commits/120f24a8aab3aaf08633bda45f1b66f584b8cde5